spec/davinci_crd_test_kit/coverage_information_system_action_across_hooks_validation_test_spec.rb

```diff
@@ -17,34 +17,18 @@
   let(:valid_coverage_info_system_action) { JSON.parse(valid_response_body)['systemActions'].first }
```

Comment:
I can't comment on the line, but this file still has a `session_data_repo` defined.